test-servers/jenkins/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jenkins:1.642.1
+FROM jenkins:latest
```

Comment:
Since this is intended only as a test utility I think latest is OK to make it convent and reduce maintenance, I found the version I was using has disappeared.  There is no risk to the project, worse that will happen is the utility won’t work..

I can pin it and maybe build a unit test but I dont think it is worth the effort..

thoughts?


On May 19, 2016, at 2:29 PM, Arnold Pelka <notifications@github.com<mailto:notifications@github.com>> wrote:


In test-servers/jenkins/Dockerfile<https://github.com/capitalone/Hygieia/pull/581#discussion_r63954615>:

> @@ -1,4 +1,4 @@
> -FROM jenkins:1.642.1
> +FROM jenkins:latest


@amitmawkin<https://github.com/amitmawkin> | @jdamick<https://github.com/jdamick> | @tabladrum<https://github.com/tabladrum> : Do we want to allow the Dockerfile to use latest Jenkins image, or does Hygieia require the specified version of the Jenkins image?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/capitalone/Hygieia/pull/581/files/51ea40229ce60b113f8b6f3cea47fd7e14632eb7#r63954615>